File: pearce_test/auditory/matlab/plotvisual.m
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% plotvisual.m
%
% Written by:  Brent Warner
% 07/02/03
%
% for use with VISUAL model
%
% loads right hemisphere *.out files in current
% directory and plots them.
%
% IT and PFC regions on Figure 2
% V1 and V4 regions on Figure 1
% 
% 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

load efd1.out
load efd2.out
load ev1h.out
load ev1v.out
load ev4c.out
load ev4h.out
load ev4v.out
load exfr.out
load exfs.out
load exss.out

figure(2)
subplot(5,1,1),plot(exss),title('EXSS')
subplot(5,1,2),plot(exfs),title('EXFS')
subplot(5,1,3),plot(efd1),title('EFD1')
subplot(5,1,4),plot(efd2),title('EFD2')
subplot(5,1,5),plot(exfr),title('EXFR')

figure(1)
subplot(5,1,1),plot(ev1h),title('EV1H')
subplot(5,1,2),plot(ev1v),title('EV1V')
subplot(5,1,3),plot(ev4h),title('EV4H')
subplot(5,1,4),plot(ev4c),title('EV4C')
subplot(5,1,5),plot(ev4v),title('EV4V')

clear all

File: visual/code/visualization/plotVisual.py
# plotVisual.py
#
# Plot output data files of visual delay-match-to-sample simulation
#
# by Antonio Ulloa, PhD
#
# Last modified on October 28 2014

import numpy as np
import pylab as pl

# Load data files
efd1 = np.loadtxt('efd1.out')
efd2 = np.loadtxt('efd2.out')
ev1h = np.loadtxt('ev1h.out')
ev1v = np.loadtxt('ev1v.out')
ev4c = np.loadtxt('ev4c.out')
ev4h = np.loadtxt('ev4h.out')
ev4v = np.loadtxt('ev4v.out')
exfr = np.loadtxt('exfr.out')
exfs = np.loadtxt('exfs.out')
exss = np.loadtxt('exss.out')

# Plot values
fig1 = pl.figure()
fig1.suptitle('EFD1')
pl.plot(efd1) 

fig2 = pl.figure()
fig2.suptitle('EFD2')
pl.plot(efd2)

fig3 = pl.figure()
fig3.suptitle('EV1H')
pl.plot(ev1h)

fig4 = pl.figure()
fig4.suptitle('EV1V')
pl.plot(ev1v)

fig5 = pl.figure()
fig5.suptitle('EV4C')
pl.plot(ev4c)

fig6 = pl.figure()
fig6.suptitle('EV4H')
pl.plot(ev4h)

fig7 = pl.figure()
fig7.suptitle('EV4V')
pl.plot(ev4v)

fig8 = pl.figure()
fig8.suptitle('EXFR')
pl.plot(exfr)

fig9 = pl.figure()
fig9.suptitle('EXFS')
pl.plot(exfs)

fig0 = pl.figure()
fig0.suptitle('EXSS')
pl.plot(exss)

# Show the plot on the screen
pl.show() 
